x-pack/plugins/event_log/server/es/context.mock.ts

```diff
@@ -17,7 +17,7 @@ const createContextMock = () => {
     logger: loggingSystemMock.createLogger(),
     esNames: namesMock.create(),
     initialize: jest.fn(),
-    waitTillReady: jest.fn(),
+    waitTillReady: jest.fn(async () => Promise.resolve(true)),
```

Comment:
micro nit.
Isn't this the same as... ?
```suggestion
    waitTillReady: jest.fn(async () => true),
```

If so, it just seems a little easier to understand and maintain.

Reply:
I should try putting that back to your suggesting - I had some issues trying to get it to fire, so tried "the long way" so I could debug it.  thx!